dotnetv3/Glacier/ListTagsForVaultExample/ListTagsForVault.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Amazon.Glacier;
+using Amazon.Glacier.Model;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ListTagsForVaultExample
+{
+    class ListTagsForVault
+    {
+        static async Task Main(string[] args)
+        {
+            var client = new AmazonGlacierClient();
+            var vaultName = "example-vault";
+
+            var request = new ListTagsForVaultRequest
+            {
+                AccountId = "-",
```

Comment:
Same as previous comment.

Reply:
As before, this is correct, so I added a comment explaining why.